### PR TITLE
Closes #3151 Update NodeJS in ProboCI config to version 20.13.0.

### LINKED ISSUE #3151: Update Probo instructions to install a more recent version of Node.
Currently our probo config is slightly out of date
https://github.com/az-digital/az_quickstart/blob/main/.probo.yaml#L35

See Latest LTS Version: 20.11.0 (includes npm 10.2.4)

## Conditions of satisfaction
- [x] Probo config installs node `20.11.0` or later. https://github.com/az-digital/az_quickstart/blob/main/.probo.yaml#L35

### PR DESCRIPTION
**Pull request by trackleft**
## Description
Updated the version that probo updates node to in .probo.yml
Updated some of the javascript packages we use in Quickstart
Updated the eslintrc config for Drupal 10.2
Updated the artifact javascript files. with `lando yarn run build`

## Related issues
Closes #3151 

## How to test
Check the probo log. https://app.probo.ci/#/dashboard/build/93a72410-486d-4ad4-ad33-f6e3c499839e/task/635ee85e19000004

Ensure `az_select_menu` and `az_ajax` still work.
Place a select menu somewhere following these instructions: https://quickstart.arizona.edu/site-admin/managing-menus/select-menu

The az_ajax js, manifests itself on this page. /pages/news-view-paragraphs
See: 

https://github.com/az-digital/az_quickstart/assets/1023167/1783aa65-4a42-455f-84b4-0079bc35e57f

### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
   - [ ] Bug fix
   - [ ] Accessibility, performance, or security improvement
   - [ ] Critical institutional link or brand change
   - [ ] Adding experimental module
   - [ ] Update experimental module
- **Minor release changes**
   - [ ] New feature
   - [ ] Breaking or visual change to existing behavior
   - [ ] Upgrade experimental module to stable
   - [ ] Enable existing module by default or database update
   - [ ] Non-critical brand change
   - [ ] New internal API or API improvement with backwards compatibility
   - [ ] Risky or disruptive cleanup to comply with coding standards
   - [ ] High-risk or disruptive change (requires upgrade path, risks regression, etc.)
- **Other or unknown**
   - [x] Other or unknown

### Drupal core

- **Patch release changes**
   - [ ] Security update
   - [ ] Patch level release (non-security bug-fix release)
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major or minor level update
- **Other or unknown**
   - [ ] Other or unknown

### Drupal contrib projects
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch or minor level update
   - [ ] Add new module
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major level update
- **Other or unknown**
   - [ ] Other or unknown

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by trackleft** [comment]
FYI https://www.drupal.org/project/drupal/issues/3371963

File: package.json [hunk 1]
```diff
@@ -27,14 +27,14 @@
     "eslint-config-prettier": "^8.4.0",
     "eslint-plugin-import": "^2.25.4",
     "eslint-plugin-jquery": "^1.5.1",
-    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-prettier": "^5.0.0",
```
Comment (joeparsons) [inline package.json:30]:
```suggestion
    "eslint-plugin-prettier": "^5.0.1",
```
Nitpick to match the constraint core uses: https://git.drupalcode.org/project/drupal/-/blob/10.2.x/core/package.json?ref_type=heads#L71

**Comment by joeparsons** [comment]
Maybe we should update the eslint config, too?
https://git.drupalcode.org/project/drupal/-/blob/10.2.x/core/.eslintrc.json

**Comment by trackleft** [comment]
Update lando file https://github.com/az-digital/az_quickstart/blob/main/.lando.yml#L50

**Comment by trackleft** [comment]
Looks like lando has a bug that does not allow installing node:20 at the moment. https://github.com/lando/node/issues/66

File: .lando.yml [hunk 2]
```diff
@@ -47,7 +47,7 @@ services:
       - composer install
       - ln -s /usr/local/quickstart-install-profile/.vscode /app/.vscode
   node:
-    type: node:18
+    type: node:20
     build:
```
Comment (trackleft) [inline .lando.yml:51]: Blocked by https://github.com/lando/node/issues/66

Comment (joeparsons) [reply .lando.yml:51]: This worked for me with the latest lando release ([v3.21.0-beta.1](https://github.com/lando/lando/releases/tag/v3.21.0-beta.1))

File: .probo.yaml [hunk 3]
```diff
@@ -32,7 +32,7 @@ steps:
       - sudo npm cache clean -f --silent --no-progress
       - sudo npm install -g --no-progress yarn
       - sudo npm install -g --no-progress n
-      - sudo n 18.12.1
+      - sudo n 20.11.0
```
Comment (trackleft) [inline .probo.yaml:35]:
```suggestion
      - sudo n 20.13.0
```

**Review by tadean — APPROVED** [review]

**Pull request merged**